docs/source/user/toc.rst

```diff
@@ -36,6 +37,7 @@ option via the *More actions* button in the table of contents toolbar. Here is a
 showing its use:
 
 .. image:: ./images/toc/numberingH1.gif
+    :alt:A GIF showing how to toggle Automatic section numbering for headings within a notebook. Clicking the button instantly applies multilevel numbering to each heading.
```

Comment:
Space is needed see https://jupyterlab--12879.org.readthedocs.build/en/12879/user/toc.html

```suggestion
    :alt: A GIF showing how to toggle Automatic section numbering for headings within a notebook. Clicking the button instantly applies multilevel numbering to each heading.
```

![Screenshot from 2022-07-31 15-02-04](https://user-images.githubusercontent.com/5832902/182029958-e4743975-9d69-428e-b34d-468428fe8064.png)